.github/workflows/deploy-artifact.yml

```
@@ -112,10 +112,10 @@ jobs:
           if ! [ -z "${{ github.event.release.tag_name }}" ] && [[ "${{ github.event.release.tag_name }}" != *"alpha"* ]]; then
             MAJOR_MINOR_VERSION=$(echo ${{ github.event.release.tag_name }} | sed -rn 's/^([0-9]+.[0-9]+).[0-9]+.*$/\1/p')
             test -z "${MAJOR_MINOR_VERSION}" && echo "::error::Tag ${{ github.event.release.tag_name }} does not adhere to semantic versioning" && exit 1
-            echo "::set-output name=branch::stable/$MAJOR_MINOR_VERSION"
-            echo "Branch = stable/$MAJOR_MINOR_VERSION"
+            echo "branch=stable/${MAJOR_MINOR_VERSION}" >> $GITHUB_OUTPUT
+            echo "Branch = stable/${MAJOR_MINOR_VERSION}"
           else
-            echo "::set-output name=branch::${{ github.event.repository.default_branch }}"
+            echo "branch=stable/${{ github.event.repository.default_branch }}" >> $GITHUB_OUTPUT
```

:x: @megglos I don't think we need the `stable` prefix here since it is related to pushes to the `main` branch, or for `alpha` releases (as per the explanation in the code comment above).

```suggestion
            echo "branch=${{ github.event.repository.default_branch }}" >> $GITHUB_OUTPUT
```

---

great catch, thanks!